package.json

```diff
@@ -35,14 +37,14 @@
     "VM"
   ],
   "dependencies": {
-    "@babel/runtime": "^7.4.0",
     "async": "^2.1.2",
     "async-eventemitter": "^0.2.2",
+    "core-js-pure": "^3.0.1",
```

Comment:
Seems to be ok as a new dependency, see [here](https://github.com/zloirock/core-js).